front/src/components/CompetitionList/Details/Positions/Positions.test.js

```diff
@@ -0,0 +1,35 @@
+import { render, screen, within } from "@testing-library/react";
+import Positions from "./Positions";
+
+describe("Positions", () => {
+    beforeEach(() => {
+        render(<Positions />);
+    });
+
+    describe("table with positions", () => {
+        let positionsTable;
+        it("should render", () => {
+            positionsTable = screen.getByRole("table");
```

Comment:
Let's make this test a little "stronger" and check that id is correct:
```suggestion
            positionsTable = screen.getByRole("table", {id: /positionsTable/i});
```